.golangci.next.reference.yml

```diff
@@ -2356,9 +2357,11 @@ linters-settings:
     for-loops: true
 
   predeclared:
-    # Comma-separated list of predeclared identifiers to not report on.
-    # Default: ""
-    ignore: "new,int"
+    # List of predeclared identifiers to not report on.
+    # Default: []
+    ignore:
```

Comment:
Should we rename?
```suggestion
    ignore-identifiers:
```

Reply:
we will stay on the name of the option from the linter.